### ISSUE TITLE
Managed Istio Ambient-based Mesh

### ISSUE THREAD

**Issue by shashankbarsin** [Azure/AKS#3803]
Managed Mesh for AKS introduces a fully managed service mesh that simplifies traffic operations by removing the need for sidecars. It offers secure-by-default connectivity, automated lifecycle management, and streamlined multi-cluster communication—reducing operational overhead and enabling scalable, resilient architectures.

**Comment by dguerin** [comment]
Great to see this on the backlog, keen to know when this feature will be GA. Thanks

**Comment by SanyaKochhar** [comment]
We will keep you posted when we have a firm timeline in place for this feature

**Comment by 3d0it** [comment]
Any updates on Ambient Mode support for the AKS managed Istio add-on?

We're currently evaluating it mainly to reduce sidecar overhead/resource usage, and it would fit our use case very well.

Thanks!

**Comment by SanyaKochhar** [comment]
@3d0it yes actually, happy to share an update - please check out our new Istio Ambient-based networking solution - Azure Kubernetes Application Network: https://learn.microsoft.com/en-us/azure/application-network/

Istio add-on will remain in sidecar mode and Application Network will be our Ambient-based offering 

cc: @kelvin-ko @dguerin
